packages/dev-approuter/lib/devApprouter.js

```diff
@@ -0,0 +1,147 @@
+const approuter = require("@sap/approuter");
+const express = require("express");
+const xsenv = require('@sap/xsenv')
+
+const findUI5Modules = require("cds-plugin-ui5/lib/findUI5Modules");
+const createPatchedRouter = require("cds-plugin-ui5/lib/createPatchedRouter");
+const applyUI5Middleware = require("cds-plugin-ui5/lib/applyUI5Middleware");
+
+const findCAPModules = require("ui5-middleware-cap/lib/findCAPModules");
+const applyCAPMiddleware = require("ui5-middleware-cap/lib/applyCAPMiddleware");
+
+const {
+    parseConfig,
+    applyDependencyConfig,
+    addDestination,
+    configureCAPRoute,
+    configureUI5Route
+} = require("./helpers");
+
+
+class DevApprouter {
```

Comment:
The reason for the class what to easily export an instance of the dev-approuter, so that dev can start it on their own if they want to instead of relying on our `index.js`. See [here](https://github.com/nicoschoenteich/ui5-ecosystem-showcase/blob/main/packages/dev-approuter/README.md#extending-the-dev-approuter). Also, the approuter works similarly with `Approuter.prototype.start = function(...) {...}`.

Reply:
Ok, thanks for the explanation